src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -176,6 +212,16 @@ throttle, strafe, rotation, getHeadingRotation2d())
       module.setDesiredState(moduleStates.get(module.getModulePosition()), isOpenLoop);
   }
 
+
+  public void setModuleStates(SwerveModuleState[] states) {
+    
+    // This needs looking at.  There might need to be some mapping, as I'm not sure of the incoming state ordering
+      SwerveModuleSparkMax[] smods = m_swerveModules.values().toArray(new SwerveModuleSparkMax[1]);
+      for(int i =0;i<states.length;i++) {
+        smods[i].setDesiredState(states[i], true);
```

Comment:
How can we test what this mapping should be?

Reply:
This feels like something that should be tested in sysid

Reply:
What's sysid?

It looks like the calling classes keep everything in order.  another MR incoming.